src/actions/socket_actions.js

```diff
@@ -0,0 +1,41 @@
+const sio = require('socket.io-client');
+const ActionTypes = require('../constants').ActionTypes;
+const PayloadSources = require('../constants').PayloadSources;
+const Dispatcher = require('../dispatcher');
+const SocketStore = require('../stores/socket_store');
+
+class SocketActions{
+    init(url){
+        var socket = sio(url);
+        socket.on("connect", this.onConnected);
+
+        Dispatcher.dispatch({
+            actionType: ActionTypes.SOCKET_INITIALIZED,
+            socket: socket
+        });
+        console.log("socket initialized");
```

Comment:
@pletcher will update the spacing. I had already made quite a few changes before I noticed the difference and changed my editor settings. The editorconfig file came down with the latest version, yes.